industrial_ci/src/util.sh

```diff
@@ -38,7 +38,7 @@
 function ici_time_start {
     if [ "$DEBUG_BASH" ] && [ "$DEBUG_BASH" == true ]; then set +x; fi
     ICI_START_TIME=$(date +%s%N)
-    ICI_TIME_ID=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
+    ICI_TIME_ID=$(date +%s%N | md5sum | fold -w 8 | head -n 1)
```

Comment:
I noticed it just after submitting the update, but I noticed that editing PR has generated another commit and I was trying to drop it.

ICI_TIME_ID=$(printf "%x\n" $ICI_START_TIME)

It does look fine for me.